### ISSUE TITLE
Unable to change alphabets to dvorak-homerow

### ISSUE THREAD

**Issue by kprav33n** [fcsonline/tmux-thumbs#74]
I tried configuring `@thumbs-alphabet` to `dvorak-homerow` like it was mentioned in the [README](https://github.com/fcsonline/tmux-thumbs#thumbs-alphabet) to use characters from Dvorak home row for selection. However, it doesn't seem to take effect.

```
$ tmux -V
tmux 3.2-rc3

$ tmux-thumbs -V
tmux-thumbs 0.5.0

$ tail -8 ~/.tmux.conf.local
# Use tmux-thumbs for quick select and paste text
set -g @thumbs-key F
set -g @thumbs-alphabet dvorak-homerow
set -g @thumbs-unique 1
set -g @thumbs-contrast 1
set -g @thumbs-command 'tmux set-buffer -w {} && tmux display-message \"Copied {}\"'
set -g @thumbs-upcase-command 'tmux set-buffer -w {} && tmux paste-buffer && tmux display-message \"Copied {}\"'
run-shell ~/.tmux/plugins/tmux-thumbs/tmux-thumbs.tmux
praveen on atom-lab-6 in ~
```

Please look at the screenshot below that still uses characters from QWERTY.

<img width="1382" alt="Screen Shot 2021-01-28 at 2 33 27 AM" src="https://user-images.githubusercontent.com/659069/106125808-71695980-6111-11eb-860a-fb0f0dcb3b1e.png">

**Comment by fcsonline** [comment]
Yep, there was a bug there. Since we split thumbs from tmux we forgot to propagate this parameter. :sweat_smile: 

Can you check this commit out and compile it? a82e46d841e44a4d01dcda1dbc209b2a957c55c3

**Comment by kprav33n** [comment]
Thanks for the swift resolution! This is fixed by a82e46d841e44a4d01dcda1dbc209b2a957c55c3. Verified with [0.5.1](../releases/tag/0.5.1).
